src/languages/en.js

```diff
@@ -77,6 +77,9 @@ export default {
         markAsUnread: 'Mark as Unread',
         editComment: 'Edit Comment',
         deleteComment: 'Delete Comment',
+        deleteConfirmation: 'Are you sure you want to delete this comment?',
+        delete: 'Delete',
+        cancel: 'Cancel',
```

Comment:
NAB but these are probably generic enough that they shouldn't be nested in `reportActionContextMenu`

Reply:
done